packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -84,6 +91,12 @@ export interface ConnectionFailedState {
   sceneViewId: string;
 }
 
+export interface ConnectionCancelledState {
+  type: 'cancelled';
+  jwtProvider?: JwtProvider;
+  sceneViewId?: string;
```

Comment:
It's not currently, just added for consistency with the `disconnected` state. Not opposed to removing it though, since I don't see as much value in the `cancelled` state